unime/src/routes/+layout.svelte

```diff
@@ -32,16 +27,15 @@
     dispatch({ type: '[App] Get state' });
   });
 
-  let showDevMode = false;
-
+  let expandDevMenu = true;
   let showDebugMessages = false;
 
-  // set color scheme
   const systemColorScheme = window.matchMedia('(prefers-color-scheme: dark)');
+
   systemColorScheme.addEventListener('change', (e) => {
     determineTheme(e.matches, $state?.active_profile?.theme);
+    systemColorScheme.removeEventListener('change', () => {});
```

Comment:
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#once), we can improve it further by supplying `{ once }` directly to the function itself, like so: `systemColorScheme.addEventListener('change', (e) => { }, { once: true });`

Reply:
Fixed